xbmc/guilib/GUIListItemLayout.cpp

```diff
@@ -176,17 +176,20 @@ void CGUIListItemLayout::LoadLayout(TiXmlElement *layout, int context, bool focu
   if (condition)
     m_condition = g_infoManager.Register(condition, context);
   m_isPlaying.Parse("listitem.isplaying", context);
-  TiXmlElement *child = layout->FirstChildElement("control");
+  // ensure width and height are valid
+  if (!m_width)
+    m_width = maxWidth;
+  if (!m_height)
+    m_height = maxHeight;
   m_group.SetWidth(m_width);
   m_group.SetHeight(m_height);
+
+  TiXmlElement *child = layout->FirstChildElement("control");
   while (child)
   {
     LoadControl(child, &m_group);
     child = child->NextSiblingElement("control");
   }
-  // ensure width and height are valid
-  m_width = std::max(1.0f, m_width);
-  m_height = std::max(1.0f, m_height);
```

Comment:
question still valid, i will merge this without behaviour change for now though.